traefik/values.yaml

```diff
@@ -165,6 +165,19 @@ persistence:
 # affinity is left as default.
 hostNetwork: false
 
+# Whether Role Based Access Control objects like roles and rolebindings should be created
+rbac:
+  # If set to false, installs ClusterRole and ClusterRoleBinding so Traefik can be used across namespaces.
+  # If set to true, installs namespace-specific Role and RoleBinding and requires provider configuration be set to that same namespace
+  namespaced: false
+  enabled: true
+
+# Whether to create the ServiceAccount
+serviceAccount:
+  create: true
```

Comment:
To me this property is not necessary.

You can handle the use case only with the name. 

If the property name is not defined or empty create the service account. otherwise reference an existing service account. 

WDYT?

Reply:
`serviceAccount.create` is a standard option supplied by `helm create`, hence why I kept it as most people are familiar with it and how to use it. Removing `serviceAccount.create` would disallow the chart creating a service account with a custom name. 

That's probably not a common use case though, so I can remove it. I'll keep the name as `serviceAccount.name` instead of `serviceAccountName` to allow the possibility to add create or other options in the future as the need arises.